enterprise/backend/src/metabase_enterprise/search/scoring.clj

```diff
@@ -2,7 +2,30 @@
   ;; TODO -- move to `metabase-enterprise.<feature>.*`
   (:require
    [metabase.public-settings.premium-features :as premium-features :refer [defenterprise]]
-   [metabase.search.in-place.scoring :as scoring]))
+   [metabase.search.config :as search.config]
+   [metabase.search.in-place.scoring :as scoring]
+   [metabase.search.postgres.scoring :as fulltext.scoring]))
+
+(def ^:private enterprise-scorers
+  {:official-collection (fulltext.scoring/truthy :official_collection)
+   :verified            (fulltext.scoring/truthy :verified)})
+
+(def ^:private scorers-enabled
+  {:official-collection #(premium-features/has-feature? :official-collection)
+   :verified            #(premium-features/has-feature? :content-verification)})
+
+(defn- additional-scorers
+  "Which additional scorers are active?"
+  []
+  (into [] (comp (filter (fn [[_ pred]] (pred))) (map first)) scorers-enabled))
```

Comment:
```suggestion
(defn- enterprise-scorers []
  (->> {:official-collection (when (premium-features/has-feature? :official-collection)
                               (fulltext.scoring/truthy :official_collection))
        :verified            (when (premium-features/has-feature? :content-verification)
                               (fulltext.scoring/truthy :verified))}
       (m/remove-vals nil?)))
```

wdyt?

Reply:
Even though your suggestion is prettier, I prefer to keep it as data. When I refactor ranker this will just be a single map with both the predicate and the expression in a value map, that just gets merged.

Reply:
Just gone ahead made it a single map, and do a dumb transformation for now.